docs/mkdocs.yml

```diff
@@ -56,7 +56,11 @@ plugins:
   - autorefs
   - redirects:
       redirect_maps:
+        'cloud/index.md': 'concepts/index.md#langgraph-platform'
+        'cloud/how-tos/index.md': 'how-tos/index.md#langgraph-platform'
         'cloud/concepts/api.md': 'concepts/index.md#langgraph-platform'
```

Comment:
api should probably map to langgraph server instead? and not to the concepts page?

Reply:
yep